questionnaires/esm28q/esm28q_armt.json

```diff
@@ -0,0 +1,1450 @@
+[
+    {
+        "field_name": "esm_sleep_well_v2",
+        "form_name": "esm28q",
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "I slept well",
+        "select_choices_or_calculations": [
+            {
+                "code": "1",
+                "label": "1 Not at all "
+            },
+            {
+                "code": "2",
+                "label": "2 "
+            },
+            {
+                "code": "3",
+                "label": "3 "
+            },
+            {
+                "code": "4",
+                "label": "4 "
+            },
+            {
+                "code": "5",
+                "label": "5 "
+            },
+            {
+                "code": "6",
+                "label": "6 "
+            },
+            {
+                "code": "7",
+                "label": "7 Very much"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "esm1_v2",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "esm_anxious_v2",
+        "form_name": "esm28q",
+        "section_header": "Think the moment just before this task",
```

Comment:
This gets inherited by the next questions as `section_header` until it is replaced, is it fine?

Reply:
Fixed